### PR TITLE
Fuzz: update dependencies

### PR DESCRIPTION
**Pull request by koushiro**
Signed-off-by: koushiro <koushiro.cqx@gmail.com>

update dependencies of fuzz

###  What is the type of the changes?

- Misc (other changes)

###  How is the PR tested?

- Manual test

###  Does this PR affect documentation (docs) or should it be mentioned in the release notes?

no

###  Does this PR affect `tidb-ansible`?

no

### CODE REVIEW

File: fuzz/cli.rs [hunk 1]
```diff
@@ -312,9 +304,8 @@ fn run_libfuzzer(target: &str) -> Result<(), Error> {
          -C incremental=fuzz-incremental \
          -C passes=sancov \
          -C llvm-args=-sanitizer-coverage-level=4 \
-         -C llvm-args=-sanitizer-coverage-trace-pc-guard \
          -C llvm-args=-sanitizer-coverage-trace-compares \
-         -C llvm-args=-sanitizer-coverage-trace-divs \
+         -C llvm-args=-sanitizer-coverage-inline-8bit-counters \
```
Comment (BusyJay) [inline fuzz/cli.rs]: Why the change?

Comment (koushiro) [reply fuzz/cli.rs]: >-fsanitize-coverage=trace-pc-guard is no longer supported by libFuzzer.
Please either migrate to a compiler that supports -fsanitize=fuzzer
or use an older version of libFuzzer

see [the ci script](https://github.com/rust-fuzz/libfuzzer/blob/master/ci/script.sh) of libfuzzer for details.

**Comment by koushiro** [comment]
@sre-bot /merge
